### ISSUE TITLE
Missing LICENSE

### ISSUE THREAD

**Issue by IBPX** [wallabag/docker#98]
This project doesn't have a license file. MIT could be used, since it has been used on other Wallabag repositories.

Before adding a LICENSE, though, permission from contributors who have contributed a significant amount is needed.

- [ ] @xsteadfastx
- [ ] @nicosomb
- [ ] @janLo
- [ ] @Strubbl
- [ ] @lotooo
- [ ] @davidmehren

If you agree, please comment,

> I agree to license all of my contributions to the **wallabag/docker** repository under the MIT license.

**Comment by janLo** [comment]
I agree to license all of my contributions to the wallabag/docker repository under the MIT license.

**Comment by nicosomb** [comment]
I agree to license all of my contributions to the wallabag/docker repository under the MIT license.

👌

**Comment by lotooo** [comment]
I agree to license all of my contributions to the wallabag/docker repository under the MIT license.

**Comment by Strubbl** [comment]
I agree to license all of my contributions to the wallabag/docker repository under the MIT license.

**Comment by davidmehren** [comment]
I agree to license all of my contributions to the wallabag/docker repository under the MIT license.
